docs/pages/machine-id/reference/configuration.mdx

```diff
@@ -3,133 +3,122 @@ title: Machine ID Configuration Reference
 description: Configuration reference for Teleport Machine ID.
 ---
 
-The following snippet shows full YAML configuration of the Machine ID client
-`tbot`. Assuming the file is written to `tbot.yaml`, it can be used with
-`tbot -c /etc/tbot.yaml`.
+This reference documents the various options that can be configured in `tbot`'s
+configuration file. This configuration file offers more control than
+configuring `tbot` using CLI parameters alone.
+
+To configure `tbot` to use a configuration file, specify the path with the `-c`
+flag, e.g `tbot -c ./tbot.yaml`.
+
+Certain terms are used in this reference and hold a specific meaning:
+
+- **Artifact**: an item that `tbot` will write to a destination as part of the
```

Comment:
Instead of a list of one term, I would rework this into a paragraph that introduces the concept of an artifact. We can make this a list again once we add more keywords.

Reply:
1acd5e732b